frontend2/package.json

```diff
@@ -15,7 +15,7 @@
     "@types/node": "^16.18.26",
     "@types/react": "^18.2.6",
     "@types/react-dom": "^18.2.4",
-    "highcharts": "^11.4.0",
+    "highcharts": "^11.4.8",
```

Comment:
ordinarily would prefer not committing npm version bumps on unrelated PRs but let's push this through because our charting is not mature yet :sunglasses: